cfme/generic_objects/definition/button_groups.py

```diff
@@ -97,13 +97,15 @@ def create(self, name, description, request, image, button_type='Default',
             'description': description,
             'display': display,
             'image': image,
-            'dialog': dialog,
             'open_url': open_url,
             'display_for': display_for,
             'request': request,
             'submit_version': submit_version,
             'system_message': system_message
         })
+        # note: for some reason this was not being filled if included in dict above
```

Comment:
Thanks for this :+1: 
John, I think better to move this as TODO so we will fix it.